apps/emqx_web_hook/src/emqx_web_hook.erl

```diff
@@ -325,7 +325,7 @@ send_http_request(ClientID, Params) ->
     {ok, Path} = application:get_env(?APP, path),
     Headers = application:get_env(?APP, headers, []),
     Body = emqx_json:encode(Params),
-    ?LOG(debug, "Send to: ~0p, params: ~0s", [Path, Body]),
```

Comment:
When Body's type is binary, [this line ]() does not print the correct log information